src/clj_kondo/impl/analyzer.clj

```diff
@@ -1372,26 +1400,9 @@
                                  (:children parsed)))]
       analyzed-expressions)
     (catch Exception e
-      (if dev? (throw e)
-          {:findings [(let [m (.getMessage e)]
-                        (if-let [[_ msg row col]
-                                 (and m
-                                      (re-find #"(.*)\[at line (\d+), column (\d+)\]"
-                                               m))]
-                          {:level :error
-                           :filename filename
-                           :col (Integer/parseInt col)
-                           :row (Integer/parseInt row)
-                           :type :syntax
-                           :message (str/trim msg)}
-                          {:level :error
-                           :filename filename
-                           :col 0
-                           :row 0
-                           :type :syntax
-                           :message (str "can't parse "
-                                         filename ", "
-                                         (or m (str e)))}))]}))
+      (if dev?
+        (throw e)
+        (->findings e filename)))
```

Comment:
I will pull this `->findings` refactor out into a separate PR to make this diff smaller.